systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2076 @@
+      description: Activate managed systems.
+      parameters:
+        - in: body
+          name: ActivateSystemsRequest
+          description: Array of system IDs to activate.
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+              example: VMware_Virtual_Platform--SN-VMware-56_4d_d2_08_ce_89_1b_70-e6_32_fc_21_6c_d0_d9_f9--MAC-00-0C-29-D0-D9-F9
+      responses:
+        200:
+          $ref: '#/responses/PartialSuccess'
+        204:
+          description: No Content
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
+  /v1/get-discovered-systems:
+    get:
+      operationId: GetDiscoveredSystems
+      x-ni-privilege: Read
+      x-ni-operation: getSystems
+      tags: [systems]
+      summary: Get discovered systems
+      description: Get network discovered systems.
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/SystemInfo'
+  /v1/get-systems-summary:
+    get:
+      operationId: GetSystemsSummary
+      x-ni-privilege: Read
+      x-ni-operation: getSystems
+      tags: [systems]
+      summary: Get systems summary
+      description: Get the number of connected and disconnected managed systems.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/ManagedSystemsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/get-jobs-summary:
+    get:
+      operationId: GetJobsSummary
+      x-ni-privilege: Read
+      x-ni-operation: getJobs
+      tags: [jobs]
+      summary: Get jobs summary
+      description: Get the number of active, failed and succeeded jobs.
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/JobsSummary'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/configure-feeds:
+    post:
+      operationId: ConfigureAndUpdateMultipleFeeds
+      x-ni-privilege: Read
+      x-ni-operation: manageFeeds
+      tags: [feeds]
+      summary: Configure and update multiple feeds
+      description: Configure and update multiple feeds.
+      parameters:
+        - in: body
+          name: FeedUris
+          required: true
+          description: Feeds to be configured and updated.
+          schema:
+            type: array
+            items:
+              type: string
+              description: The URI of the feed.
+              example: 'http://download.ni.com/support/nipkg/products/ni-package-manager/released'
+      responses:
+        200:
+          $ref: '#/responses/PartialSuccess'
```

Comment:
The `PartialSuccess` object does not contain the feeds that were successfully configured nor the feeds that failed.

Reply:
The PartialSuccess response will contain the information about the feeds that were not configured(innerErrors). I don't think we should add any additional information because:
"If any resources were created, should include a success array of objects in the response"
We are not managing feeds with this API, we are not assigning IDs to them, so I don't see them as resources in this API. Our high-level use case is to get the available packages from a list of feeds. Ideally this route and the getAvailablePackages should be merged into one single route but we decided to keep them separated(to leverage the current implementations from SMS) and merge them only if we have enough time before the release.